packages/cli/src/ui/components/InputPrompt.tsx

```diff
@@ -208,7 +209,7 @@ export const InputPrompt: React.FC<InputPromptProps> = ({
         }
       }
 
-      if (key.ctrl && key.name === 'l') {
+      if (keyMatchers.clearScreen(key)) {
```

Comment:
note that text-buffer.ts also has a large number of key bindings that should be moved to use the new system over time.